### ISSUE TITLE
No support for Parallelism

### ISSUE THREAD

**Issue by LeeMatthewHiggins** [GoogleCloudPlatform/functions-framework-dart#477]
Am I missing something? using wrong?
It looks like the current system will only handle one request at a time????

run.dart line 27
 shelf_io.serve(
Does not have shared = true
and No isolates are created

I have functions running on cloudrun that mostly call other services, it would be good to be able to process more than one at a time.

**Comment by kevmoo** [comment]
Correct, the current configuration does not (explicitly) support multiple isolates, but I/O in Dart is concurrent, so you WILL be handling (many) more than one request at a time.

But this is a good feature request!
